depends/README.md

```diff
@@ -28,7 +28,8 @@ Common `host-platform-triplet`s for cross compilation are:
 - `i686-pc-linux-gnu` for Linux 32 bit
 - `x86_64-pc-linux-gnu` for x86 Linux
 - `x86_64-w64-mingw32` for Win64
-- `x86_64-apple-darwin18` for macOS
+- `x86_64-apple-darwin18` for Intel macOS
+- `arm64-apple-darwin20` for ARM macOS
```

Comment:
Why `arm64-apple-darwin20` if `depends/config.sub arm64-apple-darwin20` returns `aarch64-apple-darwin20`?

Reply:
On second thought, I think `arm64-apple-darwin20` is the correct triplet. It corresponds with the `uname -m` output.